autoscheduler/frontend/src/components/LandingPage/SelectTerm.tsx

```diff
@@ -0,0 +1,89 @@
+import * as React from 'react';
+import {
+  Menu, MenuItem, Grid, Button,
+} from '@material-ui/core';
+
+const options = [
+  'None',
+  'Semester 1',
+  'Semester 2',
+  'Semester 3',
+  'Semester 4',
+  'Semester 5',
+  'Semester 6',
+];
+
+const ITEM_HEIGHT = 48;
+
+const SelectTerm: React.SFC = () => {
+  const [anchorEl, setAnchorEl] = React.useState<null | HTMLElement>(null);
+  const open = Boolean(anchorEl);
+  const [selectedTerm, selectTerm] = React.useState(options[0]);
+
+  const handleClick = (event: React.MouseEvent<HTMLElement>): void => {
+    setAnchorEl(event.currentTarget);
+  };
+
+  const handleClose = (option: string): void => {
+    setAnchorEl(null);
+    selectTerm(option);
+  };
```

Comment:
Add comments around here were necessary (mainly for line 19) just for explaining what these are used for / why